features/step_definitions/common_steps.rb

```diff
@@ -191,8 +201,8 @@
 end
 
 Then /I see #{MAYBE_VAR} as the value of the '(.*)' field$/ do |value, field|
-  if page.has_field?(field, {type: 'radio'}) or page.has_field?(field, {type: 'checkbox'})
-    page.find_field(field, {checked: true}).value.should == value
+  if page.has_field?(field, {type: 'radio', visible: :all}) or page.has_field?(field, {type: 'checkbox', visible: :all})
+    page.find_field(field, {checked: true, visible: :all}).value.should == value
```

Comment:
Is it worth having a wrapper around find_field, where `visible: :all` happens automatically? Seems to be a very common pattern anyway.